### ISSUE TITLE
[Main UI] Support dark mode on create account and sign-in pages

### ISSUE THREAD

**Issue by wborn** [openhab/openhab-webui#309]
Hello! I just tested the OH3 initial setup wizard and it worked well for me. :+1: 
The dark mode was also there but it is still missing from the create account and sign-in pages.
As a dark mode user it would be nice to remain in darkness as much as possible. :sunglasses:

# Create account page

![create-account](https://user-images.githubusercontent.com/12213581/90861340-e169fb80-e38b-11ea-9ddc-bfc535db9056.png)

# Sign-in page

![sign-in](https://user-images.githubusercontent.com/12213581/90861349-e464ec00-e38b-11ea-9b57-1530a46608d9.png)

**Comment by ghys** [comment]
That page is actually rendered from https://github.com/openhab/openhab-core/blob/master/bundles/org.openhab.core.io.http.auth/pages/authorize.html in openhab-core, it's not part of the main UI ;)
Currently there's no JavaScript at all, supporting the dark mode would mean we'd have to add some.

**Comment by wborn** [comment]
The OS setting would make a good default. I think if a UI wants to override the OS default, this could be accomplished by adding another parameter and provide it to the auth page in [auth-mixin.js](https://github.com/openhab/openhab-webui/blob/master/bundles/org.openhab.ui/web/src/components/auth-mixin.js)?

**Comment by ghys** [comment]
> this could be accomplished by adding another parameter and provide it to the auth page in auth-mixin.js?

Could be an option, I would say however, let's keep things simple and avoid introducing more logic if we can? (that and https://github.com/openhab/openhab-core/pull/1600 was already ready in the meantime 😉)

**Comment by wborn** [comment]
Yes the PR will be a nice improvement for dark mode users! It will probably cover most use cases since such users will probably use dark mode everywhere. :new_moon_with_face:
